android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -174,6 +176,11 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
         }
     }
 
+    @ReactProp(name = "pictureInPictureConfig")
```

Comment:
This prop pattern is new in our project right? 
It looks like it replaces the whole dispatch() logic that we use for E.g. `setCustomMessageHandlerBridgeId` am I right?

Reply:
Yes. This is different.
We use `dispatch` to perform actions - a lot of times with talking to other modules - while `ReactProp` is for passing a raw value to the underlying native view.
In this case it feels more natural to use it this way to not bloat the `'attachPlayer'` dispatch call with an additional config.

Bubbling events are also directly mapped to react props.
We even have this documentation: https://github.com/bitmovin/bitmovin-player-react-native/blob/628229bed59647f76b20e8e0c2129135eea5fb00/android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt#L130-L131

Reply:
Thanks for the explanation!